py-server/test_default_model.py

```diff
@@ -0,0 +1,38 @@
+import json
```

Comment:
nit: the convention is to name test files `XXX_test.py`, ie. `default_model_test.py`

Reply:
done